### ISSUE TITLE
NoSuchMethodError: org.apache.poi.ss.usermodel.CellStyle.getBorderBottom()S

### ISSUE THREAD

**Issue by golimarrrr** [alanhay/html-exporter#9]
I get this Error with certain input files. Is it possible to disable styles completely so I can avoid it?

**Comment by alanhay** [comment]
Hi. I have not done any work on this for a whilea nd without seeing the error I can't help. I would suggest forking and modifying yourself.

**Comment by golimarrrr** [comment]
I see there are already a few forks, do you know if there is a main one or what did those forks add or fix?

This is the error:

POI 3.17

```
java.lang.NoSuchMethodError:
org.apache.poi.ss.util.RegionUtil.setBorderBottom(IL
org/apache/poi/ss/util/CellRangeAddress;L
org/apache/poi/ss/usermodel/Sheet;L
org/apache/poi/ss/usermodel/Workbook;)V
	at uk.co.certait.htmlexporter.writer.excel.ExcelTableRowWriter.doMerge(ExcelTableRowWriter.java:54)
	at uk.co.certait.htmlexporter.writer.AbstractTableRowWriter.writeRow(AbstractTableRowWriter.java:49)
	at uk.co.certait.htmlexporter.writer.AbstractTableWriter.writeTable(AbstractTableWriter.java:39)
	at uk.co.certait.htmlexporter.writer.excel.ExcelExporter.exportHtml(ExcelExporter.java:45)
	at uk.co.certait.htmlexporter.writer.AbstractExporter.exportHtml(AbstractExporter.java:44)
```


POI 4.1.1

```
java.lang.NoSuchMethodError:
org.apache.poi.ss.usermodel.CellStyle.getBorderBottom()S
	at uk.co.certait.htmlexporter.writer.excel.ExcelTableRowWriter.doMerge(ExcelTableRowWriter.java:54)
	at uk.co.certait.htmlexporter.writer.AbstractTableRowWriter.writeRow(AbstractTableRowWriter.java:49)
	at uk.co.certait.htmlexporter.writer.AbstractTableWriter.writeTable(AbstractTableWriter.java:39)
	at uk.co.certait.htmlexporter.writer.excel.ExcelExporter.exportHtml(ExcelExporter.java:45)
	at uk.co.certait.htmlexporter.writer.AbstractExporter.exportHtml(AbstractExporter.java:44)
```

**Comment by golimarrrr** [comment]
I see on your pom.xml that you were using POI 3.9 and my project had POI 3.17 included, changed it to 3.9 and now works fine
